src/app/clusters/air-pressure-measurement-server/air-pressure-measurement-server.cpp

```diff
@@ -0,0 +1,84 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "air-pressure-measurement-server.h"
+
+#include <app/util/af.h>
+
+#include "gen/att-storage.h"
+#include "gen/attribute-id.h"
+#include "gen/attribute-type.h"
+#include "gen/cluster-id.h"
+#include "gen/command-id.h"
+
+#include <support/logging/CHIPLogging.h>
+
+#ifndef emberAfAirPressureMeasurementClusterPrintln
+#define emberAfAirPressureMeasurementClusterPrintln(...) ChipLogProgress(Zcl, __VA_ARGS__);
+#endif
+
+EmberAfStatus emberAfAirPressureMeasurementClusterGetMeasuredValue(chip::EndpointId endpoint, uint16_t * measuredValue)
+{
+    return emberAfReadServerAttribute(endpoint, ZCL_AIR_PRESSURE_MEASUREMENT_CLUSTER_ID,
+                                      ZCL_AIR_PRESSURE_MEASUREMENT_MEASURED_VALUE_ATTRIBUTE_ID, (uint8_t *) measuredValue,
+                                      sizeof(*measuredValue));
+}
+
+static EmberAfStatus emberAfAirPressureMeasurementClusterSetAltitudeCallback(chip::EndpointId endpoint, int16_t altitude)
```

Comment:
`SetAltitude`-> `SetAltitudeValue` ?

Reply:
to be consisted with `AIR_PRESSURE_MEASUREMENT_ALTITUDE` I did not name it `SetAltitudeValue`. But I can change it on both sides if requried?